actor/src/main/scala/org/apache/pekko/dispatch/BatchingExecutor.scala

```diff
@@ -67,10 +67,10 @@ private[pekko] trait BatchingExecutor extends Executor {
   // invariant: if "_tasksLocal.get ne null" then we are inside Batch.run; if it is null, we are outside
   private[this] val _tasksLocal = new ThreadLocal[AbstractBatch]()
 
-  private[this] abstract class AbstractBatch extends ArrayDeque[Runnable](4) with Runnable {
+  private[this] abstract class AbstractBatch extends util.ArrayDeque[Runnable](4) with Runnable {
     @tailrec final def processBatch(batch: AbstractBatch): Unit =
       if ((batch eq this) && !batch.isEmpty) {
-        batch.poll().run()
+        batch.pollFirst().run()
```

Comment:
isn't this the only line that needs to change?

Reply:
yes it is

Reply:
The others are lint problem

Reply:
Do you suggest me remove this?

Reply:
I think current chagne should be fine to include the lint fix .